<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExceptionsFilterDialog</class>
 <widget class="QDialog" name="ExceptionsFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exceptions Filter</string>
  </property>
  <property name="whatsThis">
   <string>&lt;b&gt;Exception Filter&lt;/b&gt;
&lt;p&gt;This dialog is used to enter the exception types, that shall be highlighted during a debugging session. If this list is empty, all exception types will be highlighted. If the exception reporting flag in the &quot;Start Debugging&quot; dialog is unchecked, no exception will be reported at all.&lt;/p&gt;
&lt;p&gt;Please note, that unhandled exceptions are always highlighted independent of these settings.&lt;/p&gt;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QListWidget" name="exceptionList">
     <property name="toolTip">
      <string>List of exceptions that shall be highlighted</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="addButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Press to add the entered exception to the list</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="exceptionEdit">
     <property name="toolTip">
      <string>Enter an exception type that shall be highlighted</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="deleteButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Press to delete the selected exception from the list</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="deleteAllButton">
     <property name="toolTip">
      <string>Press to delete all exceptions from the list</string>
     </property>
     <property name="text">
      <string>Delete All</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>261</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>exceptionList</tabstop>
  <tabstop>exceptionEdit</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>deleteAllButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExceptionsFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExceptionsFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
